src/main/resources/assets/react/documentTypes/NewOrEditDocumentType.jsx

```diff
@@ -134,121 +172,127 @@ export function NewOrEditDocumentType({
-					variables
-				})
-			}).then(response => {
-				//console.debug('response', response);
-				if (response.status === 200) {
-					if (_id) {
-						doClose();
-					} else {
-						//console.debug('response.json()', response.json()); // Promise
-						response.json().then(json => {
-							//console.debug('json', json);
-							const {
-								_id/*,
-								_name,
-								addFields,
-								properties*/
-							} = json.data.createDocumentType;
-							/*setInitialValues({ // So reset button doesn't empty all inputs
-								_name,
-								addFields,
-								properties
-							});*/
-							setId(_id);
-							setInitialValues(false); // Should unmount the EnonicForm, trigger getDocumentType, and remount Enonicform?
-						});
-					}
+				const variables = {
+					_name,
+					addFields,
+					properties
+				};
+				if (_id) {
+					variables._id = _id;
+					variables._versionKey = initialValues._versionKey;
 				}
-			});
-		}}
-		schema={SCHEMA}
-	>
-		<Modal.Content>
-			<Tab
-				defaultActiveIndex={0}
-				panes={(() => {
-					const panes = [];
-					if (_id) {
+				//console.debug('submit variables', variables);
+
+				fetch(`${servicesBaseUrl}/graphQL`, {
+					method: 'POST',
+					headers: {
+						'Content-Type':	'application/json'
+					},
+					body: JSON.stringify({
+						query: _id ? GQL_MUTATION_DOCUMENT_TYPE_UPDATE : GQL_MUTATION_DOCUMENT_TYPE_CREATE,
+						variables
+					})
+				}).then(response => {
+					//console.debug('response', response);
+					if (response.status === 200) {
+						if (_id) {
+							doClose();
+						} else {
+							//console.debug('response.json()', response.json()); // Promise
+							response.json().then(json => {
+								//console.debug('json', json);
+								const {
+									_id/*,
+									_name,
+									addFields,
+									properties*/
+								} = json.data.createDocumentType;
+								/*setInitialValues({ // So reset button doesn't empty all inputs
+									_name,
+									addFields,
+									properties
+								});*/
+								setId(_id);
+								setInitialValues(false); // Should unmount the EnonicForm, trigger getDocumentType, and remount Enonicform?
+							});
+						}
+					}
+				});
+			}}
+			schema={SCHEMA}
+		>
+			<Modal.Content>
+				<Tab
+					defaultActiveIndex={0}
+					panes={(() => {
+						const panes = [];
+						if (_id) {
+							panes.push({
+								menuItem: {
+									content: 'Fields',
+									icon: 'list',
+									key: 'fields'
+								},
+								render: () => <Tab.Pane>
+									<FieldsList
+										collectionsArr={collectionsArr}
+										globalFields={globalFields}
+										interfacesArr={interfacesArr}
+										servicesBaseUrl={servicesBaseUrl}
+									/>
+								</Tab.Pane>
+							});
+						} // if _id
 						panes.push({
 							menuItem: {
-								content: 'Fields',
-								icon: 'list',
-								key: 'fields'
+								content: 'Settings',
```

Comment:
I'm not certain what you are changing here. There was a specific requirement on how the panes should appear when creating a new or editing an old documentType. Hope you haven't broken the requirement here... Guess this is speaking to myselves aswell. Try not to change anything. Only add the feature you're supposed to add.

Reply:
I'm not changing anything. 
Code in master has two panes 'Fields' and 'Settings'.
This version has two panes. Did not touch this code.
Just git versions that gets the different panes code blocks confused.